var/spack/repos/builtin/packages/libsixel/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libsixel(MesonPackage):
+    """
+    This package provides encoder/decoder implementation for DEC SIXEL graphics, and some converter programs.
```

Comment:
@taliaferro can you break up this line into to two to fix the failing style check. (The line is >100 characters long.)

Reply:
Sure, thanks for the reminder!